pyhifiberry/audiocontrol2sio.py

```diff
@@ -0,0 +1,141 @@
+"""Socketio based API to Hifiberry OS - audiocontrol2"""
+import logging
+
+from socketio import AsyncClient, AsyncClientNamespace
+
+from .consts import DEFAULT_HOST, DEFAULT_PORT
+
+_LOGGER = logging.getLogger(__name__)
+
+class Audiocontrol2SIO:
+
+    def __init__(self, sio) -> None:
```

Comment:
`def __init__(self, sio, host=DEFAULT_HOST, port=DEFAULT_PORT) -> None:`

Reply:
Hi @willholdoway, I think these changes are not necessary. It is not used on your (HA hifiberry) side.